dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -353,23 +369,38 @@ URL findClassLocation(String className) throws ClassNotFoundException {
    * @param jarFilePaths absolute paths to jar files
    * @return map of jar file paths to classes defined in them
    */
-  private static ImmutableSetMultimap<Path, String> jarFilesToDefinedClasses(
+  @VisibleForTesting
+  static ImmutableSetMultimap<Path, String> jarFilesToDefinedClasses(
       List<Path> jarFilePaths) throws IOException {
     ImmutableSetMultimap.Builder<Path, String> pathToClasses = ImmutableSetMultimap.builder();
-
     for (Path jarFilePath : jarFilePaths) {
-      for (JavaClass javaClass : topLevelJavaClassesInJar(jarFilePath)) {
+      for (JavaClass javaClass : allJavaClassesInJar(jarFilePath)) {
```

Comment:
Not sure but could we simply use invertFrom here? https://google.github.io/guava/releases/25.0-jre/api/docs/com/google/common/collect/Multimaps.html#invertFrom-com.google.common.collect.Multimap-M-

Reply:
I couldn't find a better solution with invertFrom.

- The input for this function is a list of Paths, not Multimap.
- `ImmutableSetMultimap.flatteningToImmutableSetMultimap` seemed help but as `listAllJavaClassesInJar` throws IOException (checked exception), the lambda would look complex.